pages/glossary/interaction-states.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Interaction states
+key: interaction-states
+unambiguous: true
+objective: true
+---
+
+A condition is meet for all _interaction states_ if it is meet when the element is in each of the following states:
+
+- the element is [active][]; and
+- the element is not [active][]; and
+- the element is [visited][]; and
+- the element is not [visited][].
```

Comment:
I feel this does not make the combinations clear. Maybe we can have them explicit (since it takes the same number of lines):
```suggestion
A condition is meet for all _interaction states_ if it is meet when the element is in each of the following states:

- the element is neither [active][] nor [visited][]; and
- the element is [active][] and not [visited][]; and
- the element is [active][] and [visited][]; and
- the element is not [active][] but is [visited][].
```